Standards/scs-0211-v2-kaas-default-storage-class.md

```diff
@@ -0,0 +1,48 @@
+---
+title: SCS KaaS default storage class
+type: Standard
+status: Draft
+replaces: scs-0211-v1-kaas-default-storage-class.md 
+track: KaaS
+description: |
+  The SCS-0211 standard ensures that a default StorageClass with specific characteristics is available to KaaS users.
+---
+
+## Introduction
+
+Cluster consumers can request persistent storage via [`PersistentVolumeClaims`][k8s-pvc], which is provisioned
+automatically by cloud-provided automation.
+Storage requirements may vary across use cases, so there is the concept of storage classes (`StorageClass`).
+Storage classes define some set of storage properties and consumers can choose one of these depending on the use case.
+
+## Motivation
+
+A lot of third-party software, such as Helm charts, assume that a default storage class is configured.
+Thus, for an out-of-the-box working experience, a SCS compliant Kubernetes cluster should come
+preconfigured with a sensible default storage class providing persistent storage.
+
+## Decision
+
+A freshly provisioned Kubernetes cluster MUST have a default storage class, i.e., a `StorageClass`
+object that is annotated with `storageclass.kubernetes.io/is-default-class=true` as described in the
+[Kubernetes documentation][k8s-default-sc].
+The name of this storage class is not standardized.
+
+Users MUST be able to change the default storage class.
```

Comment:
While Kubernetes itself does not prevent that, this is not necessarily easily doable in all Kubernetes distributions. For example, Gardener does not seem to support it: https://github.com/gardener/gardener/issues/1225

The value provided by such feature is also pretty slim, as users can always just reference their own storage class.

Hence, I guess it should be dropped:

```suggestion
```

Reply:
I agree, thanks for the hint!